### PR TITLE
Light admin doc follow-up

### PR DESCRIPTION
**Pull request by pwalczysko**
This is a follow-up of https://github.com/openmicroscopy/ome-documentation/pull/1617. As the features develop, this PR is trying to adjust the doc correspondingly.

TODO:
Write a doc explaining precisely what combinations of privileges are hidden under the WebUI chackboxes and link it to this doc and vice versa.

The doc of Web UI checkboxes mapping was added in the last commit here.
This doc maps the checkboxes in the OMERO.web admin UI for the creation of the light admins to the back-end permissions, as defined by @mtbc .

To test:

  - check logic, spelling, etc.
  - check crosslinking with other docs
  - Also see OmeroWebGateway.setConfigRoles.

@joshmoore @jburel @will-moore @mtbc

### CODE REVIEW

**Comment by hflynn** [comment]
Can you add a cross reference to this new page on the server permissions page too please? I would just add it to the users section, something along the lines of "Version 5.4.0 introduces the ability to create 'Restricted Administrators' with a selected subset of permissions, see here for further information."

**Comment by pwalczysko** [comment]
All TODOs here done. Ready for review.

**Comment by pwalczysko** [comment]
``unknown document: /sysadmins/mapping-restricted-admins`` is the reason for the failure of travis here. But the document is being established in https://github.com/openmicroscopy/ome-documentation/pull/1731  - does anybody know how to solve this please ? @sbesson ?

**Comment by sbesson** [comment]
If this PR depends on the file created in #1731 as per the relative link introduced on 113d318, I would say either #1731 should be merged and `origin/develop` should be merged into this branch or both PRs should be combined into a single one.

**Comment by pwalczysko** [comment]
Went for the [second option]https://github.com/openmicroscopy/ome-documentation/pull/1726#issuecomment-321604820) suggested by @sbesson and cherry-picked here the commit from #1731 . Closing #1731 in favor of this PR and updated the header too.

File: omero/sysadmins/mapping-restricted-admins.txt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+see :doc:`/sysadmins/admins-with-restricted-privileges`.
+The OMERO.web user interface form for creation and editing of
+restricted administrators
+(see the :help:`create new users <sharing-data#admin>` section)
+collates the server-side privileges
+into fewer options and gives the options user friendly
+names. Here, the mapping of the OMERO.web options to the 
+server-side privileges is given. The server-side privileges
+are more granular and direct work with them is possible on the CLI,
+as described in :doc:`/sysadmins/cli/LightAdmins`.
```
Comment (will-moore) [inline omero/sysadmins/mapping-restricted-admins.txt]: Seems this page got renamed elsewhere. This link (and same below) are now causing build failure at https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/803/console

Comment (pwalczysko) [reply omero/sysadmins/mapping-restricted-admins.txt]: Yes, here https://github.com/openmicroscopy/ome-documentation/pull/1727/commits/456046a11d6cda280ddf199ae743f529d928fa7f - there is a bit of chicken and egg problem. Those tow PRs will not be able to live side by side.

**Comment by pwalczysko** [comment]
@will-moore : the last two commits should fix the conflict

**Comment by pwalczysko** [comment]
@will-moore Fixes the conflicts. Thanks to @sbesson

**Comment by will-moore** [comment]
Looks good to merge.

**Comment by jburel** [comment]
Thanks @will-moore and @pwalczysko 
merging so https://github.com/openmicroscopy/ome-documentation/pull/1729 can go in
@sbesson will take care of the renaming of the file

**Pull request merged**